sdk/textanalytics/Azure.AI.TextAnalytics/CHANGELOG.md

```diff
@@ -4,12 +4,24 @@
 
 ### Features Added
 
+- Added `DisplayName` property to the following classes to set the optional display name:
```

Comment:
Should we say optional display name for the job/operation so it's not confused with action name?